### PR TITLE
Added TAAR ensemble to the airflow DAG.

### PR DESCRIPTION
**Pull request by crankycoder**
WIP: do not merge

@Dexterp37 Can you take a quick look at this to see if I'm setting this up correctly?  

I've got all my code for the taar dynamodb loader in a standalone egg file.

The entry point to run the data load is over here: https://github.com/crankycoder/taar_loader/blob/master/scripts/airflow_job.py 

And the bulk of the code is a straight port of the Scala code from https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/HBaseAddonRecommenderView.scala

The new python code currently lives in my repo over at: https://github.com/crankycoder/taar_loader/blob/master/taar_loader/

### CODE REVIEW

File: jobs/ensemble_taar.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   echo "Missing arguments!" 1>&2
+   exit 1
+fi
+
+# Setup a temp directory to work in 
+mkdir tmp_ensemble
+cd tmp_ensemble
+
+# Fetch the airflow job script
+wget https://raw.githubusercontent.com/crankycoder/taar_loader/releases/1.0/scripts/airflow_job.py
```
Comment (Dexterp37) [inline jobs/ensemble_taar.sh]: I'm not sure if it's ok to load the script off a non-mozilla property

**Comment by Dexterp37** [comment]
I just left a quick comment there. I don't own this repo, but @mreid-moz should be the right person to check this PR.

**Comment by mreid-moz** [comment]
302 @sunahsuh

File: dags/longitudinal.py [hunk 2]
```diff
@@ -40,6 +40,17 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
     dag=dag)
 
+ensemble_recommender = EMRSparkOperator(
+    task_id="ensemble_recommender",
+    job_name="Extract the addon data for each Telemetry user to DynamoDB",
```
Comment (sunahsuh) [inline dags/longitudinal.py]: This parameter is actually what the EMR cluster will be named -- it'd actually be kind of tough to tie that back to the correct airflow job with this name. Please pick a name that's closer to the task_id.

File: dags/longitudinal.py [hunk 3]
```diff
@@ -40,6 +40,17 @@
...
+    execution_timeout=timedelta(hours=10),
+    instance_count=20,
+    owner="vng@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "vng@mozilla.com"],
```
Comment (sunahsuh) [inline dags/longitudinal.py]: Is there another person that could be a secondary owner for this job?

File: dags/longitudinal.py [hunk 4]
```diff
@@ -40,6 +40,17 @@
...
+    env={},
```
Comment (sunahsuh) [inline dags/longitudinal.py]: This is the object that populates the environment variables on the cluster -- looks like your script below expects `date` to be set

File: dags/longitudinal.py [hunk 5]
```diff
@@ -40,6 +40,17 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
     dag=dag)
 
+ensemble_recommender = EMRSparkOperator(
+    task_id="ensemble_recommender",
+    job_name="Extract the addon data for each Telemetry user to DynamoDB",
+    execution_timeout=timedelta(hours=10),
+    instance_count=20,
```
Comment (sunahsuh) [inline dags/longitudinal.py]: Just want to verify that this timeout and instance count make sense for this job -- does it require 20 nodes?

**Comment by acmiyaguchi** [comment]
It looks like this job was migrated to `python_mozetl/taar` and added to airflow in #243. Closing this PR.

**Pull request closed without merging**